oak_abi/src/lib.rs

```diff
@@ -139,6 +139,22 @@ extern "C" {
         handle_count: u32,
     ) -> u32;
 
+    /// Write a message to a channel using the current node's privilege.
+    ///
+    /// Write `size` bytes of data from `buf`, together with `handle_count` handles from
+    /// the space at `handle_buf`, to the channel identified by `handle`.
+    ///
+    /// Returns the status of the operation, as an [`OakStatus`] value.
+    ///
+    /// [`OakStatus`]: crate::OakStatus
```

Comment:
May be simpler to maintain by just linking to `channel_write` in the comment and explaining the difference.

Reply:
Done.